rust/sealevel/environments/mainnet3/eclipsemainnet/gas-oracle-configs-eclipsemainnet.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "domain": 1,
+        "gasOracle": {
+            "type": "remoteGasData",
+            "tokenExchangeRate": "15000000000000000000",
+            "gasPrice": "10000000000",
+            "tokenDecimals": 18
+        }
+    },
+    {
+        "domain": 1399811149,
+        "gasOracle": {
+            "type": "remoteGasData",
+            "tokenExchangeRate": "887000000000000000",
```

Comment:
sol / eth = $145 / $2456 = 0.05895865237, or is this not how it's computed?

Reply:
similar deal - this is a 50% premium on approx that value